ReleaseNotes_V1.1.txt

```diff
@@ -0,0 +1,175 @@
+
+Change Description
+==================
+
+  This new version (1.1) of the ACIS Focal Plane Thermal Model has
+  three categories of changes:
+
+    I) Create the run.dat file regardless of the value of the verbose
+       switch
+ 
+   II) Do not label the run "NOT OK" when a preference for a cold and
+       stable focal plane temperature has not been met. Also, whether
+       the preference was met or not met, the report used the term
+       "violation" This was changed to "Preference Not Met".
+
+  III) Split out ACIS-I observations from ACIS-S and assign different
+       violation temperatures to the two different categories.
+
+Rationale
+==========
+    I) It was observed that the model did not write out the run.dat file
+     for cetain values of the verbose switch.  We want the run.dat
+     file to be created for any verbosity level.
+
+   II) ACIS observations with certain characteristics have a preference
+      for a cold and stable focal plane temperature. Prior to this
+      update,  when the preference was not met the Status output by
+      the model was "NOT OK".  As this is only a preference and not a 
+      requiremnent of the observation, the output status should be
+      "OK" even if the preference is not met. Likewise, use of the
+      term "violation" was too strong. 
+
+
+  III) After much discussion, the idea of changing the violation
+       temperature of all ACIS-S observations to -112.0 deg. C was approved.
+       The violation temperature for ACIS-I observations would remain
+       unchanged at -114.0 deg. C.  Now that this change is
+       operational, the behavior of the ACIS Focal Plane Predictive
```

Comment:
Do you mean the load checker tool, not the predictive model?  The model itself doesn't care about limits.